style_links.json

```diff
@@ -71,5 +71,13 @@
   {
     "name": "Tic-Tac-Toe 2",
     "link": "Tic Tac Toe/index.html"
+  },
+  {
+    "name": "Issue Tracker",
+    "link": "Issue Tracker"
+  },
+  {
+    "name": "Password Generator",
+    "link": "Password Generator"
```

Comment:
add link to html file

Reply:
It will work. No need to add **/index.html**